frontend/src/components/Card.tsx

```diff
@@ -99,78 +98,76 @@ const Card = ({
-              <div className="flex max-w-4xl items-start justify-start pt-3">
-                {languages && (
-                  <div id="languages" className="flex flex-wrap items-center justify-normal gap-2">
-                    {languages &&
-                      languages
-                        .slice(0, visibleLanguages)
-                        .map((topic, index) => (
-                          <TopicBadge
-                            key={topic || `language-${index}`}
-                            topic={topic}
-                            tooltipLabel={`This repository uses ${topic}`}
-                            type="language"
-                          />
-                        ))}
-                    {languages && languages.length > 18 && (
-                      <button
-                        onClick={loadMoreLanguages}
-                        className="text-gray-600 dark:text-gray-300"
-                      >
-                        {toggleLanguages ? 'Show more' : 'Show less'}
-                      </button>
-                    )}
-                  </div>
-                )}
-                {topics && topics.length > 0 && (
-                  <div id="topics" className="flex flex-wrap items-center justify-normal gap-2">
-                    {topics &&
-                      topics
-                        .slice(0, visibleTopics)
-                        .map((topic, index) => (
-                          <TopicBadge
-                            key={topic || `topic-${index}`}
-                            topic={topic}
-                            tooltipLabel={`This project is labeled as "${topic}"`}
-                            type="topic"
-                          />
-                        ))}
-
-                    {topics && topics.length > 18 && (
-                      <button onClick={loadMoreTopics} className="text-gray-600 dark:text-gray-300">
-                        {toggleTopics ? 'Show more' : 'Show less'}
-                      </button>
-                    )}
-                  </div>
-                )}
-                {social && social.length > 0 && (
-                  <div id="social" className="flex items-center justify-normal gap-2">
-                    {social &&
-                      social.map((item) => (
-                        <a
-                          key={`${item.title}-${item.url}`}
-                          href={item.url}
-                          target="_blank"
-                          rel="noopener noreferrer"
-                          className="flex items-center justify-center gap-1"
-                        >
-                          <FontAwesomeIcon icon={item.icon as FontAwesomeIconProps['icon']} />
-                        </a>
-                      ))}
-                  </div>
-                )}
-              </div>
+          {(languages || topics || social) && (
```

Comment:
These can be empty arrays, so we'd better check length on them. With this there will be an empty div rendered at the bottom of the card and that does not look good.